doc/stages/filters.divider.md

```diff
@@ -34,13 +34,56 @@ This pipeline will create 10 output files from the input file readers.las.
 ]
 ```
 
+## `round_robin` Example
+
+This pipeline will create 10 output files where each consecutive point is
+inserted into a new view.
+
+```json
+[
+    "example.las",
+    {
+        "type":"filters.divider",
+        "mode":"round_robin",
+        "count":"10"
+    },
+    {
+        "type":"writers.las",
+        "filename":"out_#.las"
+    }
+]
+```
+
+## `expression` Example
+
+This pipeline will create a new view every time a point that satisfies the
+expression `UserData == 122`. It can be used in combination with `count` to
+enforce a _run_ of points that must satisfy the expression before splitting
+the view.
+
+```json
+[
+    "example.las",
+    {
+        "type":"filters.divider",
+        "expression":"UserData == 122"
+    },
+    {
+        "type":"writers.las",
+        "filename":"out_#.las"
+    }
+]
+```
+
 ## Options
 
 mode
 
-: A mode of "partition" will write sequential points to an output view until
-  the view meets its predetermined size. "round_robin" mode will iterate
-  through the output views as it writes sequential points.
+: A mode of `partition` will write sequential points to an output view until
+  the view meets its predetermined size. `round_robin` mode will iterate
+  through the output views as it writes sequential points. `expression` will
+  iterate write sequential points into an output view until the view meets
+  the expression and `count` size.
```

Comment:
Is count the overall size? Or the number of occurrences of the expression?

Reply:
`count` was dropped. See other comments.